api/src/main/java/openconsensus/OpenConsensus.java

```diff
@@ -0,0 +1,49 @@
+package openconsensus;
+
+import java.util.concurrent.atomic.AtomicReference;
+import openconsensus.trace.Tracer;
+
+/**
+ * Registry provides a global access to observability objects.
+ * An implementation is responsible for setting the Registry.
+ *
+ * @author Pavol Loffay
+ */
+public class OpenConsensus {
+
+  private static final OpenConsensus INSTANCE = new OpenConsensus();
+
+  private final AtomicReference<Tracer> tracer = new AtomicReference<>();
+
+  /**
+   * Reset to the initial state.
+   */
+  public static void reset() {
+    getInstance().tracer.set(null);
+  }
+
+  /**
+   *  Returns an instance of a {@link Tracer}.
+   *
+   * @return registered tracer or null
+   */
+  public static Tracer getTracer() {
+    return getInstance().tracer.get();
```

Comment:
we discussed yesterday with DynaTrace that it may be useful to have named tracers registry. Not sure if they end up filing issue for this

Reply:
Filed an issue :) https://github.com/bogdandrutu/openconsensus/issues/223